UKHomeOffice/cerberus-service
# COP-9945: Add tests to verify Task version date/time

## Description
Add tests to verify the following 2 scenarios,

GIVEN a task exists in the task list
WHEN an update is received and a new version is created
THEN the datetime of the update is displayed in the version header

GIVEN a task exists in the task list
WHEN an update is received and a new version is NOT created
THEN the datetime of the latest version is updated to the current datetime in the version header

## To Test
npm run cypress:runner
run these 2 tests from `task-version-details.spec.js`

`Should verify datetime of the latest version when new version is not created`
`Should verify datetime of the update is displayed in the version header when new version is created`

## Developer Checklist

\* Required
